stdlib/public/core/StringUTF16View.swift

```diff
@@ -201,6 +201,17 @@ extension String.UTF16View: BidirectionalCollection {
       return _foreignIndex(i, offsetBy: n)
     }
 
+    if n.magnitude <= _StringBreadcrumbs.breadcrumbStride {
```

Comment:
On average, using breadcrumbs ought to win when |n| > breadcrumbStride/2, right?

Reply:
Yep, the average cost of using breadcrumbs is breadcrumbStride/2 each time we do it. In this case, we're doing it twice, though, so I expect the cutoff to be doubled. 🤔